paddle/framework/executor.cc

```diff
@@ -132,8 +112,8 @@ void Executor::Run(const ProgramDescBind& pdesc, Scope* scope, int block_id,
   }
 }
 
-Executor::Executor(const platform::DeviceContext& device)
-    : device_contexts_({&device}), own_(false) {}
+// Executor::Executor(const platform::DeviceContext& device)
```

Comment:
Why comment these lines? I think this ctor is still useful.

Reply:
This interface should be removed, because we only need the place indicator. I will rewrite how the while_op, recurrent_op using executor in next PR.

Reply:
I see. 

Before rewriting ops that need to run executor, please take a look at https://github.com/PaddlePaddle/Paddle/issues/6664